### PR TITLE
Disable unpublish with a flag.

### PR DESCRIPTION
**Pull request by steveworley**
- Adds a flag to `deploy` to disable the automatic unpublish of routes

### CODE REVIEW

File: src/commands/deploy.js [hunk 1]
```diff
@@ -29,6 +29,12 @@ command.builder = (yargs) => {
     type: 'boolean',
     default: false,
   });
+  yargs.options('disable-unpublish', {
+    describe: 'Disable the automatic cleanup process',
```
Comment (steveworley) [inline src/commands/deploy.js:33]:
```suggestion
    describe: 'Skip the automatic cleanup process',
```

File: src/commands/deploy.js [hunk 2]
```diff
@@ -29,6 +29,12 @@ command.builder = (yargs) => {
     type: 'boolean',
     default: false,
   });
+  yargs.options('disable-unpublish', {
```
Comment (steveworley) [inline src/commands/deploy.js:32]:
```suggestion
  yargs.options('skip-unpublish', {
```

File: src/commands/deploy.js [hunk 3]
```diff
@@ -88,6 +94,11 @@ command.handler = async function(argv) {
     console.log(chalk.bold.green('✅') + ` ${filepath}`);
   });
 
+  if (argv['disable-unpublish']) {
+    console.log(chalk.yellow('[skip]: Skipping automatic path removal.'));
```
Comment (steveworley) [inline src/commands/deploy.js:98]:
```suggestion
    console.log(chalk.yellow('[skip]: Skipping automatic cleanup'));
```

**Comment by steveworley** [comment]
Fixes #71

**Review by stooit — APPROVED** [review]

**Pull request merged**